### DISCUSSION TITLE
Is there a new App Store Connect webhook?

### DISCUSSION THREAD

**Discussion by rogerluan** [fastlane/fastlane#16897] [Ideas] [answered]
I haven't found any, but I thought asking the community :) Basically it'd be great to achieve what this plugin does without polling Apple's APIs 😁 https://github.com/erikvillegas/itunes-connect-slack

**Comment by max-ott** [comment, marked as answer]
You will have to poll Apple. The "old" plugin does the same. There's no webhook from Apple. We always have to query and compare / check for changes.

**Reply by rogerluan** [reply]
Sad 😢 I'll check if there's an open request to add this to Apple's APIs, and create one if there isn't 👍 thanks for the prompt reply!

**Comment by yannisalexiou** [comment]
Apple introduced official support for App Store Connect Webhooks at WWDC 2025, including events related to the app review process (e.g., Ready for Review, Rejected, Approved). You can find more details [here](https://developer.apple.com/documentation/appstoreconnectapi/creating_and_configuring_webhooks).

If you’d like to forward these events to tools like Slack or Microsoft Teams, I created an open-source webhook proxy that receives Apple’s notifications and formats them into readable messages.

➡️ [GitHub Repo – App Store Webhook Proxy](https://github.com/yannisalexiou/appstore-webhook-proxy)
